### ISSUE TITLE
Request: Add function to validate new account to User Management section

### ISSUE THREAD

**Issue by Shteve0** [TheNAF/naflm#13]
Currently only accessible via automated email?

**Comment by williamleonard** [comment]
How do you want this done?  Do you want to browse to the specific coach or do you want to go to a similar page that you go to through the email?

**Comment by hutchinsfairy** [comment]
You could add them to the coachlist section with an "inactive" status and a link to activate? That way you could see all pending new users in one place and not have to rely on the email for anything other than notification. For non-admins I guess you would still only want to show activated coaches.

**Comment by williamleonard** [comment]
Sounds good.  The only odd thing may be that when I created the registration module, I used the retired field as the field to activate.  Also just for future reference of other modules, since the registration module is a module, I will checked that the module is enabled but add this code to the core code.

**Comment by Shteve0** [comment]
Yeah, just moving the activate coach function into somewhere you can get to it without going via email (ideally as another box on the user management page) would be good.  A major value add would be a print list of yet to be activated accounts immediately beneath it.  Will, are you saying that's just a list of retired coaches?

**Comment by rythos42** [comment]
Really like hutchinsfairy's plan of adding a list of pending users. The notification e-mails are nice, but needing to manually type in a username is error-prone and could get lost really easily.

**Comment by rythos42** [comment]
I added a table to the admin User Management section that shows all users with "retired = 2" and a button to activate. I also changed the admin notification e-mail to direct to the User Management page.

The only thing left I want to change is that when you click Activate, it redirects to the Registration module Activate page. I'd like if the activation came from User Management, that it redirects back to User Management. The code got more complex, and I gotta run!
